src/SFA.DAS.FAA.Web/Models/VacanciesViewModel.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Newtonsoft.Json;
+using SFA.DAS.FAA.Domain.SearchResults;
+using SFA.DAS.FAT.Domain.Interfaces;
+using SFA.DAS.FAT.Web.Services;
+
+namespace SFA.DAS.FAA.Web.Models;
+
+public class VacanciesViewModel
+{
+    private readonly IDateTimeService dateTimeService;
+
+    public VacanciesViewModel(IDateTimeService dateTimeService)
+    {
+        this.dateTimeService = dateTimeService;
+    }

```

Comment:
Need to rethink this - using the implicit operator probably isnt correct when we want to start doing thins like this